$OpenBSD: patch-docs_CMakeLists_txt,v 1.1 2020/07/07 17:33:56 rsadowski Exp $

ninja: error: '/usr/ports/pobj/keepassxc-2.6.0/keepassxc-2.6.0/docs/topics/*',
needed by 'docs/KeePassXC_GettingStarted.html', missing and no known rule to
make it

Index: docs/CMakeLists.txt
--- docs/CMakeLists.txt.orig
+++ docs/CMakeLists.txt
@@ -26,16 +26,16 @@ set(OUT_DIR ${CMAKE_CURRENT_BINARY_DIR})
 # Build html documentation on all platforms
 add_custom_command(OUTPUT KeePassXC_GettingStarted.html
     COMMAND ${ASCIIDOCTOR_EXE} -D ${OUT_DIR} -o KeePassXC_GettingStarted.html ${DOC_DIR}/GettingStarted.adoc
-    DEPENDS ${DOC_DIR}/topics/* ${DOC_DIR}/styles/* ${DOC_DIR}/images/* ${DOC_DIR}/GettingStarted.adoc
+    DEPENDS ${DOC_DIR}/GettingStarted.adoc
     VERBATIM)
 add_custom_command(OUTPUT KeePassXC_UserGuide.html
     COMMAND ${ASCIIDOCTOR_EXE} -D ${OUT_DIR} -o KeePassXC_UserGuide.html ${DOC_DIR}/UserGuide.adoc
-    DEPENDS ${DOC_DIR}/topics/* ${DOC_DIR}/styles/* ${DOC_DIR}/images/* ${DOC_DIR}/UserGuide.adoc
+    DEPENDS ${DOC_DIR}/UserGuide.adoc
     WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
     VERBATIM)
 add_custom_command(OUTPUT KeePassXC_KeyboardShortcuts.html
     COMMAND ${ASCIIDOCTOR_EXE} -D ${OUT_DIR} -o KeePassXC_KeyboardShortcuts.html ${DOC_DIR}/topics/KeyboardShortcuts.adoc
-    DEPENDS ${DOC_DIR}/topics/KeyboardShortcuts.adoc ${DOC_DIR}/styles/*
+    DEPENDS ${DOC_DIR}/topics/KeyboardShortcuts.adoc
     VERBATIM)
 
 add_custom_target(docs ALL DEPENDS KeePassXC_GettingStarted.html KeePassXC_UserGuide.html KeePassXC_KeyboardShortcuts.html)
@@ -50,11 +50,9 @@ install(FILES
 if(APPLE OR UNIX)
     add_custom_command(OUTPUT keepassxc.1
         COMMAND ${ASCIIDOCTOR_EXE} -D ${OUT_DIR} -b manpage ${DOC_DIR}/man/keepassxc.1.adoc
-        DEPENDS ${DOC_DIR}/man/*
         VERBATIM)
     add_custom_command(OUTPUT keepassxc-cli.1
         COMMAND ${ASCIIDOCTOR_EXE} -D ${OUT_DIR} -b manpage ${DOC_DIR}/man/keepassxc-cli.1.adoc
-        DEPENDS ${DOC_DIR}/man/*
         VERBATIM)
     add_custom_target(manpages ALL DEPENDS keepassxc.1 keepassxc-cli.1)
 
